apps/cnspec/cmd/backgroundjob/serve_windows.go

```diff
@@ -0,0 +1,114 @@
+//go:build windows
+// +build windows
+
+package backgroundjob
+
+import (
+	"time"
+
+	"github.com/rs/zerolog/log"
+	"go.mondoo.com/cnquery/logger/eventlog"
+	"golang.org/x/sys/windows/svc"
+	"golang.org/x/sys/windows/svc/debug"
+)
+
+func Serve(timer time.Duration, handler JobRunner) {
+	isIntSess, err := svc.IsAnInteractiveSession()
+	if err != nil {
+		log.Fatal().Err(err).Msg("failed to determine if we are running in an interactive session")
+	}
+	// if it is an service ...
+	if !isIntSess {
+		// set windows eventlogger
+		w, err := eventlog.NewEventlogWriter(SvcName)
+		if err != nil {
+			log.Fatal().Err(err).Msg("failed to connect to windows event log")
+		}
+		log.Logger = log.Output(w)
+
+		// run service
+		runService(SvcName, false, timer, handler)
```

Comment:
nit: Do we need to parse a const as parameter?

Reply:
I agree, we can simplify that one.